### PR TITLE
Update Spanish translations

### PR DESCRIPTION
**Pull request by ibito**
Just updating missing translations, sorry >.<

### CODE REVIEW

File: config/locales/es.json [hunk 1]
```diff
@@ -19,7 +19,14 @@
             "remove": "¡No se está ignorando mas a {user}!"
         },
         "lang": {
-            "title": "Idiomas Disponibles   💬"
+            "success": "Se cambió el lenguaje a `{language}`!  {flag}",
+            "title": "Available languages   💬",
```
Comment (markokajzer) [inline config/locales/es.json:23]: This is Englando!

**Review by markokajzer — APPROVED** [review]
This is pretty epic man!! Thank you so much!

Can you adjust that one key? Seems like that slipped under the radar?

**Comment by markokajzer** [comment]
Feel free to add yourself to the Readme! At the bottom there is a section with acknowledgements :)

File: config/locales/es.json (hunk 1 above)
Comment (ibito) [reply config/locales/es.json]: Just fixed this 🤦‍♂️
Yes it slipped under the radar 🤣

**Comment by markokajzer** [comment]
Thanks again so much and sorry for the long wait. I was feeling a bit burned out over the holidays.

Happy New Year!

**Pull request merged**